apps/learning-snippets/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@itwin/presentation-learning-snippets",
+  "version": "0.0.0",
+  "description": "Learning snippets for ReadMes",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "build:watch": "tsc -w",
+    "clean": "rimraf lib build",
+    "cover": "npm run -s test",
+    "lint": "eslint ./src/**/*.{ts,tsx}",
+    "test": "mocha --config ./.mocharc.json",
+    "docs": "betools extract --fileExt=ts,tsx --extractFrom=./src --recursive --out=./build/docs/extract"
+  },
+  "dependencies": {
+    "@itwin/tree-widget-react": "workspace:*",
+    "@itwin/presentation-hierarchies": "^0.5.0",
+    "@itwin/presentation-shared": "^0.4.1",
+    "@itwin/itwinui-react": "3.14.1",
+    "@itwin/unified-selection": "^0.5",
+    "@itwin/appui-abstract": "^4.8.0",
+    "@itwin/appui-react": "^4.16.0",
+    "@itwin/build-tools": "^4.8.0",
+    "@itwin/components-react": "^4.16.0",
+    "@itwin/core-backend": "^4.8.0",
+    "@itwin/core-bentley": "^4.8.0",
+    "@itwin/core-common": "^4.8.0",
+    "@itwin/core-frontend": "^4.8.0",
+    "@itwin/core-geometry": "^4.8.0",
+    "@itwin/core-orbitgt": "4.8.0",
+    "@itwin/core-quantity": "^4.8.0",
+    "@itwin/core-react": "^4.16.0",
+    "@itwin/core-telemetry": "^4.8.0",
+    "@itwin/ecschema-metadata": "^4.8.0",
+    "@itwin/ecschema-rpcinterface-common": "^4.8.0",
+    "@itwin/ecschema-rpcinterface-impl": "^4.8.0",
+    "@itwin/eslint-plugin": "^4.1.1",
+    "@itwin/imodel-components-react": "^4.16.0",
+    "@itwin/presentation-backend": "^4.8.0",
+    "@itwin/presentation-common": "^4.8.0",
+    "@itwin/presentation-components": "^5.3.3",
+    "@itwin/presentation-frontend": "^4.8.0",
+    "@itwin/presentation-testing": "^5.0.13",
+    "@testing-library/dom": "^9.3.3",
+    "@testing-library/react": "^14.1.2",
+    "@testing-library/user-event": "^14.5.2",
+    "@types/chai": "4.3.5",
+    "@types/chai-as-promised": "^7.1.1",
+    "@types/chai-jest-snapshot": "^1.3.4",
+    "@types/chai-subset": "1.3.3",
+    "@types/jsdom": "^21.1.6",
+    "@types/mocha": "^10.0.6",
+    "@types/node": "^18.18.10",
+    "@types/react": "^18.0.34",
+    "@types/react-dom": "^18.0.11",
+    "@types/sinon": "^17.0.2",
+    "@types/sinon-chai": "^3.2.12",
+    "@typescript-eslint/eslint-plugin": "^7.16.1",
+    "@typescript-eslint/parser": "^7.16.1",
+    "chai": "^4.3.7",
+    "chai-as-promised": "^7.1.1",
+    "chai-jest-snapshot": "^2.0.0",
+    "chai-subset": "1.6.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-react": "^7.34.4",
+    "eslint-plugin-unused-imports": "^3.2.0",
+    "fast-xml-parser": "^4.3.6",
+    "global-jsdom": "^9.2.0",
+    "ignore-styles": "^5.0.1",
+    "jsdom": "^23.1.0",
+    "mocha": "^10.2.0",
+    "raf": "^3.4.0",
+    "react": "^18.0.34",
+    "react-dom": "^18.0.34",
+    "rimraf": "^6.0.1",
+    "sinon": "^17.0.1",
+    "sinon-chai": "^3.7.0",
+    "source-map-support": "^0.5.6",
```

Comment:
`source-map-support` can be dropped as `mocha` has support for source maps https://github.com/iTwin/presentation/blob/64527df9a542c52dd42d0d6d5f7e95ccea699431/packages/components/package.json#L39

Reply:
Fixed